src/Cli/func/Common/Utilities.cs

```diff
@@ -53,6 +53,19 @@ internal static void PrintVersion()
                 .WriteLine($"Function Runtime Version: {ScriptHost.Version}\n".DarkGray());
         }
 
+        internal static void WarnIfPreviewVersion()
+        {
+            if (!Constants.CliVersion.Contains(Constants.PreviewVersionSuffixLabel, StringComparison.OrdinalIgnoreCase))
+            {
+                return;
+            }
+
+            ColoredConsole
+                .WriteLine("You are running a preview version of Azure Functions Core Tools.".DarkYellow())
+                .WriteLine("In this preview 'python' and 'powershell' worker support is not available for linux-arm64.".DarkYellow())
```

Comment:
maybe: `In this preview, worker support for 'python' and 'powerShell' is not available on linux-arm64.`

Reply:
How about something like this? "This version currently doesn't support linux-arm64 with Python or PowerShell workers."

Do we have any ability to determine if we're on that architecture for this line? I think we always want the preview line in this state, but if I use the preview on another architecture, I don't think we need the second line.

Reply:
Hmm that's a good idea - let me see if that's possible.